### PR TITLE
Following FreeDesktop specification

### PR DESCRIPTION
**Pull request by KSXGitHub**
addressing https://github.com/gi-lom/gnabel/pull/5#discussion_r467883448 and https://github.com/gi-lom/gnabel/pull/5#discussion_r467886965

### CODE REVIEW

**Comment by KSXGitHub** [comment]
cc @fushinari

File: gnabel.py [hunk 1]
```diff
@@ -17,7 +17,8 @@
 LanNumber = 8 #number of language tuples to save in history
 ButtonLength = 65 #length of language buttons
 ButtonNumLanguages = 3 #number of language buttons
-SettingsFile = os.path.expanduser('~/.config/gnabel/settings.json')
+XdgConfigHome = GLib.get_user_config_dir()
+SettingsFile = os.path.join(XdgConfigHome, 'gnabel/settings.json')
```
Comment (fushinari) [inline gnabel.py:21]: `os.path.join(XdgConfigHome, 'gnabel/settings.json')` should be `os.path.join(XdgConfigHome, 'gnabel', 'settings.json')`

Comment (KSXGitHub) [reply gnabel.py:21]: I guess they are different on Windows? Then again, does `GLib.get_user_config_dir()` work on Windows?

Comment (fushinari) [reply gnabel.py]: Yes, it does

File: gnabel.py [hunk 2]
```diff
@@ -14,13 +14,13 @@
 from pydub import AudioSegment
 from pydub.playback import play
 
-#Constant values
-MaxLength = 1000 #maximum number of characters you can translate at once
-TransNumber = 10 #number of translations to save in history
-LanNumber = 8 #number of language tuples to save in history
-ButtonLength = 65 #length of language buttons
-ButtonNumLanguages = 3 #number of language buttons
-XdgConfigHome = GLib.get_user_config_dir()
```
Comment (KSXGitHub) [inline gnabel.py]: @gi-lom It seems that you had accidently deleted this line. You should add it back for your application to work.

**Pull request merged**